.vscode/settings.json

```diff
@@ -45,6 +45,14 @@
       "fileMatch": ["/test-assets/sfdx-simple/force-app/main/default/waveTemplates/**/layout.json"],
       "url": "/packages/analyticsdx-template-lint/src/schemas/layout-schema.json"
     },
+    {
+      "fileMatch": ["/packages/analyticsdx-template-lint/test/unit/schemas/testfiles/readiness/**/*.json"],
```

Comment:
This hooks up readiness-schema.json locally to file in the dev vscode, to make it easier when making test readiness files.